python-sdk/tests_integration/astro_deploy/master_dag.py

```diff
@@ -122,6 +122,7 @@ def start_sftp_ftp_services_method():
     instance_id = instance[0].instance_id
     ti = get_current_context()["ti"]
     ti.xcom_push(key=EC2_INSTANCE_ID_KEY, value=instance_id)
+    time.sleep(120)  # Need to delay to this to prevent error of unknown instance_id.
```

Comment:
Could we have more details on this error `instance_id`? Why/when does this happen..? Why are we setting 120s?